ethereum/contracts/query/QueryResponse.sol

```diff
@@ -115,7 +140,8 @@ abstract contract QueryResponse {
     uint8 public constant QT_ETH_CALL_BY_TIMESTAMP = 2;
     uint8 public constant QT_ETH_CALL_WITH_FINALITY = 3;
     uint8 public constant QT_SOL_ACCOUNT = 4;
-    uint8 public constant QT_MAX = 5; // Keep this last
```

Comment:
If we expect this list to continue to grow, we could consider using a custom type (e.g., Enum).